### ISSUE TITLE
New translation: Chinese (Taiwan)

### ISSUE THREAD

**Issue by civalin** [squidfunk/mkdocs-material#895]
## Translation (`zh-TW`)

``` jinja
{% macro t(key) %}{{ {
  "language": "zh-TW",
  "clipboard.copy": "複製",
  "clipboard.copied": "已複製",
  "edit.link.title": "編輯此頁",
  "footer.previous": "上一頁",
  "footer.next": "下一頁",
  "meta.comments": "留言",
  "meta.source": "來源",
  "search.language": "jp",
  "search.placeholder": "搜尋",
  "search.result.placeholder": "打字進行搜尋",
  "search.result.none": "沒有符合的項目",
  "search.result.one": "找到 1 個符合的項目",
  "search.result.other": "找到 # 個符合的項目",
  "search.tokenizer": "[\，\。]+",
  "skip.link.title": "跳轉到",
  "source.link.title": "前往倉庫",
  "toc.title": "本頁目錄"
}[key] }}{% endmacro %}
```

Originally, I try to smoothing the `zh-Hant` translation, but according the discuss in #788, it may more suitable to be placed into a individual translation file.

Thanks so much!

**Comment by squidfunk** [comment]
Taiwanese is actually not part of the HTML5 language codes, see https://www.w3schools.com/tags/ref_language_codes.asp. Currently, the selected language will also be set as `<html lang={language}...>`, so this would result in the HTML being invalid. Is there some other, similar code we can use?

**Comment by civalin** [comment]
Thanks reply, I think using code `zh-Hant` is good enough for this purpose.

If I should do some further task, please let me know :)

**Comment by squidfunk** [comment]
zh-Hant is currently already associated with Traditional Chinese:

https://github.com/squidfunk/mkdocs-material/blob/8217dcd8cd81b5ace534bd8f04199f262e565987/src/partials/language/zh-Hant.html#L23-L43

**Comment by martinbira** [comment]
According to [IANA Language Sub-tag Registry](https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry) the TW sub-tag is for the language Twi... However, TW is a correct region sub-tag for Taiwan, but not as a HTML5 language code. I've checked the [BCP 47](https://tools.ietf.org/rfc/bcp/bcp47.txt) and TW is only mentioned as a region, not a specific language. So to keep the HTML valid I guess the `zt-Hant` language code is preferred unless `zt-Hant-TW` can be used.

**Comment by civalin** [comment]
Hello @squidfunk, I know `zh-Hant` are already exists. But, if it would not cause conflict in you system, maybe allow two translation files (`zh-Hant.html`, `zh-TW.html`) generate the same `lang` attribute (`zh-Hant`) in `<html>` tag is not a bad idea. After all, Taiwanese usually think they are using "Traditional Chinese".

There are still some word choices are different from current `zh-Hant` implementation. Just like the difference between British English and American English. It just ... strange or unfamiliar? So a Taiwanese may consider it's some non-ideal translation, and try to "fix" it, just like #788 try to do.

Btw, acturally, current `zh-Hant` implementation also not a "regional-free"  implementation of Traditional Chinese (I think it more like Hong Kong accent, or a machine translation from Simplified Chinese because the word choices are very similar). For example, can we image a "English" neither British nor American even nor AnyRegion, only appear in a official grammar book? It's imposible, just like that.

If the name `zh-Hant` already be occupied by some regional-habit, and refuse further "tweak" (It's reasonable if maintainer want provide regional support). Taiwanese have no choice to build a new translation file.

But, If maintainer don't want to provide regional support, please consider accept #788 or something like that, or else any improvement for current translation are imposible. Because, basically, a native speaker barely know how to distinct the difference of "a bad translation" and "just not fit my accent".

I know the maintainability is very important, I will accept any your final decision.

**Comment by civalin** [comment]
Sorry, It too hard for me to make a regional-free version due to lacking enough cross-region linguistic knowledge. and afraid would cause some unwanted controversy.

**Comment by squidfunk** [comment]
Released as part of 3.0.6.
